org.eclipse.scout.rt.rest.test/src/test/java/org/eclipse/scout/rt/rest/client/proxy/ErrorDoRestClientExceptionTransformerTest.java

```diff
@@ -56,6 +56,24 @@ public void testTransformStatusSuccessful() {
     assertEquals(mockCause, m_exceptionTransformer.transform(mockCause, mockResponse(Status.OK)));
   }
 
+  @Test
+  public void testWithoutErrorDo() {
+    String message = "mock";
+    RuntimeException mockCause = new RuntimeException(message);
```

Comment:
Add another testcase for defaultTransform(), e.g. using a server error like Status.INTERNAL_ERROR